Description: Shows remaining recording capacity in recordings menu.
 (Requires timer-info patch)
Author: Walter Koch <koch@u32.de>

Index: vdr-1.7.26/menu.c
===================================================================
--- vdr-1.7.26.orig/menu.c	2012-03-11 21:47:00.072013037 +0100
+++ vdr-1.7.26/menu.c	2012-03-11 21:47:00.102013037 +0100
@@ -2590,6 +2590,19 @@
          }
       }
   free(LastItemText);
+  
+//kw
+  char *buffer = NULL;
+  int FreeMB;
+  int Percent = VideoDiskSpace(&FreeMB);
+  int Minutes = int(double(FreeMB) / MB_PER_MINUTE);
+  int Hours = Minutes / 60;
+  Minutes %= 60;
+  asprintf(&buffer, "%s   %d%%  %2d:%02d %s  (%.1f GB)", tr("Disk"), Percent, Hours, Minutes, tr("free"), FreeMB / 1024.0 );
+  SetTitle(buffer);
+  free(buffer);
+//kw
+  
   Refresh |= SetFreeDiskDisplay(Refresh);
   if (Refresh)
      Display();
